includes/admin/class-sensei-learners-main.php

```diff
@@ -471,18 +472,24 @@ protected function get_row_data( $item ) {
 				 * This filter runs on the learner management screen for a specific course.
 				 * It provides the learner row column details.
 				 *
-				 * @param array $columns{
-				 *   type string $title
-				 *   type string $date_started
-				 *   type string $course_status (completed, started etc)
-				 *   type html $action_buttons
+				 * @param string tag
+				 * @param array $columns {
```

Comment:
Some of those columns don't match what is being passed in the array. To save some back and forth, it should look something like this:

```
* @param array $columns {
*   @type string $title            Learner name (HTML).
*   @type string $date_started     Course start date.
*   @type string $date_completed   Course completion date.
*   @type string $user_status      Course status.
*   @type string $enrolment_status Enrollment status.
*   @type html $actions            Actions that can be taken for this learner.
* }
* @param object $item      Current WP_Comment item.
* @param int $post_id      Course ID.
* @param string $post_type Post type.
```

Not sure if the linter will complain about some alignment issues with the above or not.